addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,739 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amazonechocontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="account">
+		<label>Amazon Account</label>
+		<description>Amazon Account where your amazon echo is registered.</description>
+		<config-description>
+			<parameter name="amazonSite" type="text" required="true">
+				<default>
+				</default>
+				<label>

```

Comment:
Do you really need all those newlines, would the following do:

```xml
<label>Amazon site</label>
``` 

Depending on the implementation it could even have a different meaning

Reply:
Apply everywhere in this XML's

Reply:
Not the newlines are not needed, the was added by a edtor, because I tried to replace the blanks with tabs, but unfortunately this editor have added the newlines, I will remove them.

Reply:
OK, changed